src/doc/markdown/page120_working_with_jaia_software.md

```diff
@@ -0,0 +1,119 @@
+# Working With Jaia Software
+
+### Table of Contents:
+- [Launching The Simulator](#launching-the-simulator)
+- [Modifying Code](#modifying-code)
+- [Deploying Code](#deploying-code)
+
+<br>
+<a id="launching-the-simulator"></a>
+
+# Launching the Simulator
+Note: The Jaia simulator works __exclusively with Ubuntu 20.04__
+1. Clone the jaiabot repo (https://github.com/jaiarobotics/jaiabot)
+2. Run the setup scripts
+```
+cd /path/to/jaiabot/scripts
+./setup-tools-build.sh
+./setup-tools-runtime.sh
+```
+3. Run the build script
+```
+cd /path/to/jaiabot
+./build.sh
+```
+4. Launch the JCC web interface
+```
+cd /path/to/jaiabot/src/web
+./run.sh
+```
+5. Launch the simulator in a separate terminal
+```
+cd /path/to/jaiabot/config/launch/simulation
+# Set the simulation to run 4 bots at a time warp of 5
+./generate_all_launch.sh 4 5
+./all.launch
+```
+6. Troubleshooting the simulator
+* Kill all processes
+```
+# Kill all processes, then relaunch the simulator
+cd /path/to/jaiabot/scripts
+./kill-jaiabot-processes.sh
+```
+
+* Refresh the build directory
+
+```
+# Remove the build directory
+cd /path/to/jaiabot
+rm -rf build
+
+# Re-create the build directory
+./build.sh
+```
+
+<br>
+<a id="modifying-code"></a>
+
+# Modifying Code
+1. [Launch the simulator](#launching-the-simulator)
+2. Modify the code as you see fit
+3. Shutdown the simulator and JCC web interface
+```
+# Shutdown the simulator
```

Comment:
These instructions might confuse an inexperienced user, I think.  We need to specify to them to press Ctrl-C inside the terminal where the programs are running, rather than cd-ing to the directory and pressing Ctrl-C.  Obvious to us, but maybe not to the audience of this document?

Reply:
Yes, great point